examples/kustomize-inflate-helm/kustomize/kustomization.yaml

```diff
@@ -0,0 +1,12 @@
+resources:
+  - secret.yaml
+
+helmGlobals:
+  chartHome: .
+
+helmCharts:
+  # Must match the directory name with the unpacked chart
```

Comment:
can you add a note that this is pointing to the wordpress.tar chart tarball as well?  It's not immediately clear